apps/design-system/registry/default/example/page-layout-detail.tsx

```diff
@@ -0,0 +1,136 @@
+
+      <PageContainer size="large">
+        <PageSection orientation="horizontal">
+          <PageSectionMeta>
+            <PageSectionSummary>
+              <PageSectionTitle>Subscription</PageSectionTitle>
+              <PageSectionDescription>
+                View and manage your current subscription plan and billing cycle.
+              </PageSectionDescription>
+            </PageSectionSummary>
+          </PageSectionMeta>
+          <PageSectionContent>
+            <Card>
+              <CardContent className="p-6">
+                <div className="space-y-4">
+                  <div>
+                    <p className="text-xs text-foreground-lighter mb-1">Current Plan</p>
+                    <p className="text-sm font-medium">Pro Plan</p>
+                  </div>
+                  <div>
+                    <p className="text-xs text-foreground-lighter mb-1">Billing Cycle</p>
+                    <p className="text-sm">Monthly</p>
+                  </div>
+                  <div>
+                    <p className="text-xs text-foreground-lighter mb-1">Next Billing Date</p>
+                    <p className="text-sm">March 15, 2024</p>
+                  </div>
+                  <div className="pt-2">
+                    <Button type="default" size="small">
+                      Change Plan
+                    </Button>
+                  </div>
+                </div>
+              </CardContent>
+            </Card>
+          </PageSectionContent>
+        </PageSection>
+
+        <PageSection orientation="horizontal">
+          <PageSectionMeta>
+            <PageSectionSummary>
+              <PageSectionTitle>Cost Control</PageSectionTitle>
+              <PageSectionDescription>
+                Set spending limits and alerts to manage your organization's costs.
+              </PageSectionDescription>
+            </PageSectionSummary>
+          </PageSectionMeta>
+          <PageSectionContent>
+            <Card>
+              <CardContent className="p-6">
+                <div className="space-y-4">
+                  <div>
+                    <p className="text-xs text-foreground-lighter mb-1">Monthly Spending Limit</p>
+                    <p className="text-sm">$500.00</p>
+                  </div>
+                  <div>
+                    <p className="text-xs text-foreground-lighter mb-1">Current Month Spend</p>
+                    <p className="text-sm">$234.50</p>
+                  </div>
+                  <div className="pt-2">
+                    <Button type="default" size="small">
+                      Configure Limits
+                    </Button>
+                  </div>
+                </div>
+              </CardContent>
+            </Card>
+          </PageSectionContent>
+        </PageSection>
+
+        <PageSection orientation="horizontal">
+          <PageSectionMeta>
+            <PageSectionSummary>
+              <PageSectionTitle>Payment Methods</PageSectionTitle>
```

Comment:
Minor nit: capitalisation across titles and buttons. Sentence case for all?

Just thinking folks might use this as gospel, so would be good to match however we decide to capitalise these things.